src/Parser/EnhancementShaman/Modules/ShamanCore/Rockbiter.js

```diff
@@ -0,0 +1,37 @@
+import SPELLS from 'common/SPELLS';
+
+import Module from 'Parser/Core/Module';
+
+const MAELSTROM_THRESHOLD = 95;//120 is threshold, but energize event values are after the 25 Maelstrom increase is applied
+
+class Rockbiter extends Module {
+  rockbiterOveruse = [];
+  maelstromWasted = 0;
+
+  on_byPlayer_energize(event) {
+    if(event.ability.guid === SPELLS.ROCKBITER.id) {
+      if(event.classResources[0].amount >= MAELSTROM_THRESHOLD) {
+        this.rockbiterOveruse.push(event);
+        this.maelstromWasted+= event.waste;
+      }
+    }
+  }
+
+  on_finished() {
+    console.log("maelstromWasted " + this.maelstromWasted);
+    console.log(this.rockbiterOveruse);
+  }
+
+  suggestions(when) {
```

Comment:
I'd prefer if this suggestion checked to a number that is about the same on all fights length. Wasting 100 maelstrom is much worse in 3 minutes than in 15 minutes.